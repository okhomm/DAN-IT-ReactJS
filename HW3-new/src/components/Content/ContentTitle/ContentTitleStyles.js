import styled from 'styled-components'

export const StyledTitle = styled.h2`
  position: relative;
  padding-left: 26px;
  color: #3c4242;
  font-family: 'Core Sans C', sans-serif;
  font-size: 34px;
  font-style: normal;
  font-weight: 600;
  line-height: 33.5px;
  letter-spacing: 0.68px;

  &::before {
    content: '';
    position: absolute;
    left: 0;
    top: 0;
    width: 6px;
    height: 30px;
    border-radius: 10px;
    background-color: #8a33fd;
  }
`;