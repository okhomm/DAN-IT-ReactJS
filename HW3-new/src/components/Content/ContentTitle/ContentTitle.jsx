import { StyledTitle } from './ContentTitleStyles'

const ContentTitle = ({ children }) => {
  return <StyledTitle>{children}</StyledTitle>
}

export default ContentTitle
