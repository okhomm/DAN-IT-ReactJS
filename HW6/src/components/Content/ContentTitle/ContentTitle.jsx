import React from 'react'
import { StyledTitle } from './ContentTitleStyles'

const ContentTitle = ({ children }) => {
  return <StyledTitle>{children}</StyledTitle>
}

export default ContentTitle
