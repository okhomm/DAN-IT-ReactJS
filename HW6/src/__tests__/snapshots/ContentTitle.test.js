import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import ContentTitle from '../../components/Content/ContentTitle/ContentTitle';

test('renders ContentTitle component', () => {
  const { container } = render(<ContentTitle>Test Title</ContentTitle>);
  
  expect(container).toMatchSnapshot();
});
