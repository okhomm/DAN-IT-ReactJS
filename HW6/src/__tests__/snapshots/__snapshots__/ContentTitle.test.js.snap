// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ContentTitle component 1`] = `
<div>
  <h2
    class="sc-bdnyFh lauAjk"
  >
    Test Title
  </h2>
</div>
`;
